EKrut-Server/src/ekrut/server/db/InventoryItemDAO.java

```diff
@@ -181,4 +237,48 @@ public boolean updateItemQuantity(int itemId, int quantity, String ekrutLocation
 			}
 		}
 	}
+
+	/**
+	 * Fetches the quantity of an item from the inventory system.
+	 */
+	public class ItemQuantityFetcher implements iItemQuantityFetcher {
```

Comment:
There doesn't appear to be any good reason why this is an inner class. Consider changing this to a static nested class or even a non-nested class (I'm more inclined to choose the latter).